spark/tests/test_spark.py

```diff
@@ -424,13 +428,15 @@ def proxy_with_warning_page_mock(url, *args, **kwargs):
     'spark.job.num_skipped_stages': 8000,
     'spark.job.num_failed_stages': 9000,
 }
-
-SPARK_JOB_SUCCEEDED_METRIC_TAGS = [
+SPARK_JOB_SUCCEEDED_METRIC_TAGS_NO_ID = [
```

Comment:
Can we rename these to `SPARK_JOB_SUCCEEDED_METRIC_TAGS_NO_STAGE_ID`? Just to make it clearer that they refer to no `stage_id` tags and not the `job_id` tag.

Reply:
Sure!